### ISSUE TITLE
But can it run under ARM

### ISSUE THREAD

**Issue by tsadigovAgmail** [anbox/anbox#2111]
Simple plain question
Can I use it on ARM based machine?
I have a SBC I plan to use as daily driver.
I want it to replace my laptop and phone
Can I use anbox to make my ARM sbc run Android at the same time it is running Ubuntu?

**Comment by Fuseteam** [comment]
no it cannot run on arm hardware, furthermore anbox is unfinished alpha software it is not reliable as a daily driver

but yes if it could it could indeed be used to "run ubuntu and android at the same time".

if you run ubuntu 22.04 or newer; anbox's conceptual succesor, waydroid, could suit your needs. It is also alpha software, so i wouldn't rely on it, but it does run on arm hardware

**Comment by schrmh** [comment]
Maybe I'm misunderstanding something here but it is possible to use Anbox on a phone that uses a ARM processor...
Several people used it on mobile distros like postmarketOS

**Comment by Fuseteam** [comment]
no, mobile linux distros have no dropped anbox in favor of waydroid

**Comment by Little-Elephant** [comment]
@schrmh is there any sort of how-to?

**Comment by schrmh** [comment]
@Little-Elephant Well I guess you have to figure it out from pages like 
https://github.com/anbox/anbox/issues/1357#issuecomment-593106461  
https://docs.ubports.com/en/latest/userguide/dailyuse/anbox.html  
https://wiki.postmarketos.org/wiki/Anbox  
https://gitlab.com/postmarketOS/pmaports/-/issues/17  
Or ask in the Matrix rooms related to mobile linux.

**Comment by Fuseteam** [comment]
Seriously, Anbox development on postmarketos and ubuntu touch is now dead. They have moved on to an alternative

**Comment by anoraktrend** [comment]
This is because anbox sucks

**Comment by Fuseteam** [comment]
That is just plain rude, without anbox the alternative would not exist
